src/blueprint/from_config/operators/graphql.rs

```diff
@@ -21,10 +21,10 @@ pub fn update_graphql<'a>(
         graphql.base_url.as_ref().or(config.upstream.base_url.as_ref()),
         "No base URL defined".to_string(),
       )
-      .zip(helpers::headers::to_headermap(&graphql.headers))
-      .and_then(|(base_url, header_map)| {
+      .zip(helpers::headers::to_headervec(&graphql.headers))
+      .and_then(|(base_url, headers)| {
         Valid::from(
-          RequestTemplate::new(base_url.to_owned(), operation_type, &graphql.name, args, header_map)
+          RequestTemplate::new(base_url.to_owned(), operation_type, &graphql.name, args, headers)
```

Comment:
@meskill Please help me understand is this really required?

Reply:
I've refactored some shared code to reuse in grpc operator and to make it more clear had to rename some entities

Reply:
to explain changes from header_map to vec of mustaches: I just put that logic in shared function to reuse for different operators since before templates for headers were created inside request template, look https://github.com/tailcallhq/tailcall/pull/730/files#diff-875bcc31c837288db9085e8d55a0b5ae4f97d493ef644a05df513731bd3b8464L82-L85 and https://github.com/tailcallhq/tailcall/pull/730/files#diff-f3bdd9f02462ede38765771f8eb5d93eae1575ff2a4bec0a50c02c0d72df7fb0L107-L111